torch/tensortypes/nestedtensor/nested.py

```diff
@@ -0,0 +1,215 @@
+import torch
```

Comment:
`torch.tensortypes.nestedtensor` is such a mouthful. Why not just `torch.nested`?

Reply:
Sure, fine by me